turbopack/crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -848,16 +848,14 @@ pub(crate) async fn analyse_ecmascript_module_internal(
         .resolved_cell();
         analysis.set_async_module(async_module);
     } else if let Some(span) = top_level_await_span {
-        AnalyzeIssue {
-            code: None,
-            message: StyledString::Text("top level await is only supported in ESM modules.".into())
-                .resolved_cell(),
-            source_ident: source.ident(),
-            severity: IssueSeverity::Error.resolved_cell(),
-            source: Some(issue_source(*source, span)),
-            title: ResolvedVc::cell("unexpected top level await".into()),
-        }
-        .cell()
+        AnalyzeIssue::new(
+            IssueSeverity::Error.cell(),
+            source.ident(),
+            Vc::cell("unexpected top level await".into()),
+            StyledString::Text("top level await is only supported in ESM modules.".into()).cell(),
+            None,
+            Some(issue_source(*source, span)),
+        )
         .emit();
```

Comment:
This breaks persistent caching as only resolved cells are allows to be emitted.

Reply:
Should I revert this PR?

Reply:
I'll check if inverting to make it resolved is possible.